rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -605,7 +613,7 @@ public JobResultData jobResult(@HeaderParam("sessionid") String sessionId, @Path
     @Override
     public JobInfoData jobInfo(String sessionId, String jobId)
             throws NotConnectedRestException, PermissionRestException, UnknownJobRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)